frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTable.java

```diff
@@ -100,7 +100,6 @@ public GitMacheteGraphTable(Project project) {
 
     setDefaultRenderer(BranchOrCommitCell.class, BranchOrCommitCellRendererComponent::new);
 
-    setCellSelectionEnabled(false);
```

Comment:
Hmm maybe instead explicit `setCellSelectionEnabled(true);`?

Reply:
Hmm, this seems to be an unintended side effect:
![Screenshot from 2020-07-31 12-14-10](https://user-images.githubusercontent.com/56402877/89026988-f4a81f00-d329-11ea-8270-eb5cecd7bad4.png)
Now rows have blue selection marker again

Reply:
Explained via _Slack_.
We bring it back. TBD what to do with commit cells